frontend/src/scenes/session-recordings/player/SessionRecordingPlayer.tsx

```diff
@@ -27,9 +27,17 @@ export function useFrameRef({
     return frame
 }
 
-export function SessionRecordingPlayerV2({ sessionRecordingId, playerKey }: SessionRecordingPlayerProps): JSX.Element {
-    const { handleKeyDown } = useActions(sessionRecordingPlayerLogic({ sessionRecordingId, playerKey }))
-    const { isSmallScreen } = useValues(sessionRecordingPlayerLogic({ sessionRecordingId, playerKey }))
+export function SessionRecordingPlayerV2({
+    sessionRecordingId,
+    playerKey,
+    recordingStartTime, // While optional, including recordingStartTime allows the underlying ClickHouse query to be much faster
+}: SessionRecordingPlayerProps): JSX.Element {
```

Comment:
It doesn't look like we're passing in this prop anywhere we use this component. I think it'd be best if we either omit this prop in the V2 or implement it in `SessionPlayerDrawer` where it's used.